devsetup/Makefile

```diff
@@ -9,6 +9,11 @@ EDPM_NETWORK_CONFIG_TEMPLATE ?= templates/single_nic_vlans/single_nic_vlans.j2
 EDPM_SSHD_ALLOWED_RANGES ?= ['192.168.122.0/24']
 EDPM_CHRONY_NTP_SERVER ?= pool.ntp.org
 EDPM_PLAY_CRD ?= edpm/edpm-play.yaml
+CLEANUP_DIR_CMD ?= rm -Rf
+
+define vars
+${1}: export CLEANUP_DIR_CMD=${CLEANUP_DIR_CMD}
+endef
```

Comment:
This function vars is not called anywhere. Can we drop this?

Reply:
oh, right. hmm. dumb me copy-pasting things. I'll amend it shortly.

Reply:
It's now called in the needed places.